docs/tutorial/vald-agent-standalone-on-k8s.md

```diff
@@ -0,0 +1,373 @@
+1. [Running exapmle code](#Running-Exapmle-Code)
+1. [Cleanup](#Cleanup)
+
+## Requirements
+
+- kubernetes: v1.19 ~
+- go: v1.15 ~
+- helm: v3 ~
+- libhdf5 (_only required for get started_)
+
+Helm is used to deploying Vald on your kubernetes and Hdf5 is used to decode the sample data file to run the example.<br>
+If helm or hdf5 is not installed, please install [helm](https://helm.sh/docs/intro/install) and [hdf5](https://www.hdfgroup.org/).
+
+<details><summary>Installatoin command for helm</summary><br>
+
+```bash
+curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+```
+
+</details>
+
+<details><summary>Installation command for hdf5</summary><br>
+
+```bash
+# yum
+yum install -y hdf5-devel
+
+# apt
+apt-get install libhdf5-serial-dev
+
+# homebrew
+brew install hdf5
+```
+
+</details>
+
+## Prepare the kubernetes cluster
+
+1. Prepare kubernetes cluster
+
+    To complete get started, the kubernetes cluster is required.<br>
+    Vald will run on Cloud Service such as GKE, AWS.
+    In the sense of trying to "Get-Started", [k3d](https://k3d.io/) or [kind](https://kind.sigs.k8s.io/) are easy kubernetes tools to use.
+
+## Deploy Vald Agent Standalone on Kubernetes Cluster
+
+This chapter will show you how to deploy a standalone Vald Agent using Helm and run it on your kubernetes cluster. <br>
+This chapter uses [NGT](https://github.com/yahoojapan/ngt) as Vald Agent to perform vector insertion operation, indexing and searching operations.<br>
+
+1. Clone the repository
+
+    To use the `deployment yaml` for deploy, let's clone [`vdaas/vald`](https://github.com/vdaas/vald.git) repository.
+
+    ```bash
+    git clone https://github.com/vdaas/vald.git
+    cd vald
+    ```
+
+1. Deploy Vald Agent Standalonw using Helm
+
+    There is the [values.yaml](https://github.com/vdaas/vald/blob/master/example/helm/values-standalone-agent-ngt.yaml) to deploy standalone Vald Agent.
+    Each component can be disabled by setting the value `false` to the `[component].enabled` field.
+    This is useful for deploying standalone Vald Agent NGT pods.
+
+    ```bash
+    helm repo add vald https://vald.vdaas.org/charts
+    helm install vald-agent-ngt vald/vald --values example/helm/values-standalone-agent-ngt.yaml
+    ```
+
+1. Verify
+
+    ```bash
+    kubectl get pods
+    ```
+
+    <details><summary>Example output</summary><br>
+    If the deployment is successful, Vald Agent component should be running.
+
+    ```bash
+    NAME               READY   STATUS    RESTARTS   AGE
+    vald-agent-ngt-0   1/1     Running   0          20m
+    vald-agent-ngt-1   1/1     Running   0          20m
+    vald-agent-ngt-2   1/1     Running   0          20m
+    vald-agent-ngt-3   1/1     Running   0          20m
+    ```
+
+    </details>
+
+## Running Exapmle Code
+
+1. Port Forward
+
+    At first, port-foward is required to make request from your local environment possible.
```

Comment:
```suggestion
    At first, port-foward is required to make a request from your local environment possible.
```

Reply:
I think `make request` is correct

Reply:
```suggestion
    At first, port forwarding the vald-lb-gateway is required to make request to the Vald cluster from your local environment.
```

Reply:
`to the Vald cluster` is not necessary for this sentence 🤔